Assets/Sequence/SequenceSDK/WaaS/EOAWalletLinker.cs

```diff
@@ -0,0 +1,62 @@
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Sequence.WaaS
+{
+    public class EOAWalletLinker
+    {
+        private static string NonceGenerationLink =
+            "https://demo-waas-wallet-link-server.tpin.workers.dev/generateNonce";
```

Comment:
Can you make this configurable? We shouldn't embed our demo server in the SDK. Then from Jelly Forest implementation you can pass this URL.

Reply:
Done, it is now passed in via the constructor.
Do we have a doc on how devs can configure one of these servers?

Reply:
Nice, thanks. Yep we have one here: https://docs.sequence.xyz/solutions/wallets/link-wallets/integration-guide